workload_profiling/src/test/scala/org/apache/spark/sql/rapids/tool/profiling/ApplicationInfoSuite.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.tool.profiling
+
+import java.io.{File, PrintWriter}
+
+import com.nvidia.spark.rapids.tool.profiling.ProfileArgs
+import org.scalatest.FunSuite
+import scala.collection.mutable.ArrayBuffer
+
+import org.apache.spark.internal.Logging
+
+class ApplicationInfoSuite extends FunSuite with Logging {
+
+  val sparkSession = ProfileUtils.createSparkSession
+  var apps :ArrayBuffer[ApplicationInfo] = ArrayBuffer[ApplicationInfo]()
+  val appArgs = new ProfileArgs(Array("src/test/resources/eventlog_minimal_events"))
+
+  test("test single event") {
+    val tempFile = new File("src/test/resources/workload_profiling")
```

Comment:
this should create a file with random name, this seems to be putting it in the src tree here?

you can use Java File.createTempFile 
spark also has a Utils.createTempDir but looks like its private